projects/core/src/i18n/testing/mock-translate.ts

```diff
@@ -4,10 +4,14 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-export function mockTranslate(key: string | undefined, options: any = {}) {
+export function mockTranslate(
+  key: string | string[] | undefined,
+  options: any = {}
+) {
+  const keyString = Array.isArray(key) ? key[0] : key;
```

Comment:
This function's purpose is being a simple but comprehensive and bullet-proof util for testing to be used by many libs. To keept it so, what about printing all keys:
```suggestion
Array.isArray(key) ? key.join(',') : key;
```
?

(I didn't include space after comma deliberately, because other options are separated with spaces)

Reply:
Sure, good catch!

Reply:
I've also added 2 tests covering the scenario array of keys. to `mock-translate.spec.ts`